pkg/apic/apiserver/clients/management/v1alpha1/ExternalSecret.go

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
There is another PR raised for updating the model to include ExternalSecret needed by governance agents. We should sync with the changes once that PR is merged.

Reply:
Merged the other PR if you want to sync those changes over @sbolosan